node/lib/util/rebase_util.js

```diff
@@ -403,6 +439,34 @@ up-to-date.`);
                     inits.push(initRebaser.init(id.tostrS()));
                 }
                 else {
+                    if (SubmoduleConfigUtil.modulesFileName === e.path) {
+
+                        // If there is a conflict in the '.gitmodules' file,
+                        // attempt to resolve it by comparing the current
+                        // change against the original onto commit and the
+                        // merge base between the base and onto commits.
+
+                        const next = yield metaRepo.getCommit(rebaseOper.id());
+                        const Conf = SubmoduleConfigUtil;
+                        const getSubs = Conf.getSubmodulesFromCommit;
+                        const fromNext = yield getSubs(metaRepo, next);
+                        const fromBase = yield getMergeBaseSubs();
+                        const fromOnto = yield getOntoSubs();
+                        const merged = Conf.mergeSubmoduleConfigs(fromNext,
+                                                                  fromOnto,
+                                                                  fromBase);
+                        // If it was resolved, write out and stage the new
+                        // modules state.
+
+                        if (null !== merged) {
```

Comment:
I think that if I do nothing there, libgit2 will have the merge markers in place. Will verify.